javascript-modal/styles.css

```diff
@@ -0,0 +1,73 @@
+/* Default styles */
+
+.open-btn {
+  width: 240px;
+  height: 58px;
+  background-color: #4a4ae9;
+  color: #fff;
+  font-size: 32px;
+  font-family: 'Open Sans', sans-serif;
+  padding: 6px;
+  border-radius: 6px;
+  box-shadow: 3px 3px 3px #919292;
+}
+
+.pop-container {
+  display: none;
+}
+
+/* Clicked mode styles */
+
+.background-clicked {
+  background-color: #323232;
+}
+
+.pop-container-clicked {
+  display: flex;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.open-btn-clicked {
+  width: 240px;
+  height: 58px;
+  background-color: #110f2e;
+  color: #222231;
+  font-size: 32px;
+  font-family: 'Open Sans', sans-serif;
+  padding: 6px;
+  border-radius: 6px;
+  border: none;
+}
+
+/* End dark mode styles, begin pop-up styles */
+
+.pop-up {
+  position: absolute;
+  left: 1000px;
+  top: 500px;
```

Comment:
You want to avoid giving such high px values. If you set all the offset values to 0, then it would cover the whole page (which is what we want)